New handler

mhowe opened the pull request.

mhowe committed:
New handler

ChadRehmKineticData reviewed (commented):
Matt, are you confident that error will always come back as a JSON object?  This just bit me the other day.  Sometimes the API returned a html string that would fail when JSON.parse was done against it.

ChadRehmKineticData committed:
Added generic handler

ChadRehmKineticData committed:
updated names in files

ChadRehmKineticData committed:
Updated handler for new api call

mhowe committed:
New Uploading an Attachment versions to support Kinetic Platform

This was the _v1 handler modified to use attachment data from the Kinetic Platform

mhowe committed:
Updated the add comment handler to handle errors that are not JSON

mhowe commented:
Updated the handler if the errors can't be parsed with JSON.

ChadRehmKineticData reviewed (approved):
looks good
